### PR TITLE
ID key removal from "stored"

### PR DESCRIPTION
**Pull request by asimon91**
This PR fixes an small bug that causes an exception when trying to make a patch call.

Example:

PATCH call URL: project.task.work/25
Body: {"name": "foobar"}

Without the fix, backend will raise an exception (id key not found).

With the fix, the task work with ID 25 is successfully patched.

### CODE REVIEW

**Comment by ecarreras** [comment]
Can you try to send the body as:

```json
{
  "id": 25,
  "name": "foobar"
}
```

I think the system was designed to have the `id` indeed the body and then detect it is an update

**Comment by asimon91** [comment]
@ecarreras it raises an exception too (list index out of range).

Which would be the best way to make PATCH calls, passing the ID on URL and body, or only on one of them?

File: backend/utils.py [hunk 1]
```diff
@@ -79,6 +79,8 @@ def recursive_crud(model, values):
         item_id = vals.pop('id', None)
         if vals:
             stored = model.read([item_id], list(vals.keys()))[0]
+            if 'id' in stored.keys():
+                stored.pop('id', None)
```
Comment (ecarreras) [inline backend/utils.py]: If you use `pop('id', None)` you don't need to check if the key exists. Simplt do `stored.pop('id', None)`

**Comment by ecarreras** [comment]
@asimon91 you're right, I think there is an issue. Only make the change commented and I'll merge it! Thank you so much! :cake:

**Comment by asimon91** [comment]
@ecarreras Fixed!

**Pull request merged**